wpilibj/src/main/java/edu/wpi/first/wpilibj/CameraServer.java

```diff
@@ -503,7 +504,7 @@ public UsbCamera startAutomaticCapture(int dev) {
    * Start automatically capturing images to send to the dashboard.
    *
    * @param name The name to give the camera
-   * @param dev The device number of the camera interface
+   * @param dev  The device number of the camera interface
```

Comment:
Opinion: I disagree with this tagging style.  I think that one space between the tag name and the description is fine.